pages/index.tsx

```diff
@@ -117,8 +117,8 @@ const App = ({ name, numberOfTejass }) => {
   );
 };
 
-App.getInitialProps = () => {
-  const numberOfTejass = 14;
+App.getInitialProps = async () => {
```

Comment:
Why?

Reply:
That was an attempted merge of a change ahead of mine. I thought you did it.